data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -21,8 +21,9 @@
         id=Kaleh
         type=Quenoth Youth
         canrecruit=yes
-        gold=200
-        {INCOME 6 4 2}
+        {GOLD 220 180 140}
+        {INCOME 4 3 2}
+        village_gold=1
```

Comment:
Please give the player a hint about the lowered value of villages.

Reply:
Hmm, I'm pretty sure that's unnecessary since the income is displayed at the top anyway.

Reply:
i agree with @stevecotton here, since the top bar does only displays the current income, not the per village income. But isnt 1 the default anyways ?

Reply:
The default is 2. I don't think it's necessary because the per village income isn't really vital for a player to know. If they really want to, they can look at scenario settings to see it, but the real stat that matters for strategy is your aggregate income from all sources.

At least to me, I think that making a special note about village income is the same thing as making a special note about base income (which is also unnecessary).